requirements.dev.txt

```diff
@@ -23,3 +23,4 @@ httpx-oauth
 httpx
 asgi_lifespan
 uvicorn
+pycryptodome
```

Comment:
Could we save us this dependency? I wouldn't mind having hard-coded key strings in the unit tests.

Reply:
Done - I've hardcoded these in `test_authentication_strategy_jwt.py` instead.